docs/Creating-Your-Own-EmulationStation-Theme.md

```diff
@@ -1,166 +1,153 @@
-*['Spare' Theme Detailed View]*
+<figure markdown>
+  ![Spare Theme Detailed View](https://i.imgur.com/r5SHwkc.jpg)
+  <figcaption>Spare Theme Detailed View</figcaption>
+</figure>
 
 It is a simple, clean theme, not very outlandish, but it's enough to teach the essentials, and a few of the more advanced ideas.
 
----
+**Note**
 
-## Notes
+> This tutorial was developed on Windows, but all the main parts (the XML and images) can be done on any Operating System, as long as you can put or assess the files into the `theme` folder on your Raspberry Pi (e.g., with Samba, see [setup](#use-samba-share-of-emulationstations-themes-on-retropie)).
 
-I made this tutorial on a Windows PC, but all the main parts (the XML and images) can be done on any Operating System, and as long as you can FTP or SSH into the 'theme' folder on your Raspberry Pi, then you can test your theme there.
-
-**What You Will Need**
+### What You Will Need
 
 - A text editor
-    - Notepad will do fine, but something with syntax highlighting is helpful. I use Sublime Text 3, but others
-like Notepad++ are good too.
-- Emulationstation
-    - I use a version on my PC (more on that below), but the version on your Pi is fine.
+    - Notepad will do fine, but something with syntax highlighting is way more helpful. Whatever editor you prefer: It should support XML formatting and maybe even XML linting.
+- EmulationStation
+    - This tutorial uses a version on a Windows machine (more on that below), but the version on your Pi is fine.
 - An image editor
-    - This is optional as all the images will be supplied, however you can use these images as a base to create your own, or make your own from scratch. I use Photoshop, but other programs like GIMP or Paint.net would work too.
+    - This is optional as all the images will be supplied, however you can use these images as a base to create your own, or make your own from scratch. On Windows you may use Photoshop, but other programs like GIMP, Inkscape, Krita or Paint.net would work too. Whenever possible use SVG format for your graphics instead of pixel based formats (JPG/PNG) as the former scales properly without visible artifacts.
 - The ES Theme Helper by @Rookervik
     - Not essential, but very, very handy
     -  [Get it from @Rookervik's DropBox here](https://dl.dropboxusercontent.com/u/60872572/EmulationStation/ES%20Theme%20Helper3.rar)
 - Any coding experience
-    - Again, not essential, but it makes things easier. To be honest, as long as you know that `<tag>` opens a tag and `</tag>` closes a tag, then you should be fine.
-
-There will be a list of all links and resources at the end of the tutorial.
+    - Again, not essential, but it makes things easier. To be honest, as long as you know that `<tag>` opens a tag (more precisely an XML element) and `</tag>` closes a tag, then you should be fine.
 
----
+The (bogus) files you will need for this tutorial are listed [here](#references).
```

Comment:
I'd prefer _sample_ or _testing_ instead of _bogus_. The files themselves are not bogus, they're bugs ROMs/games.

Reply:
Done.